fonll.sh

```diff
@@ -0,0 +1,38 @@
+package: FONLL
+version: "1.3.3"
+tag: "v1.3.3"
+source: https://github.com/brinick/fonll.git
+requires:
+  - lhapdf
+---
+#!/bin/bash -ex
+
+make ${JOBS+-j$JOBS} -f $SOURCEDIR/main/Makefile VPATH="$SOURCEDIR/main:\
+                                                        $SOURCEDIR/hdmassive:\
+                                                        $SOURCEDIR/hdresummed:\
+                                                        $SOURCEDIR/phmassive:\
+                                                        $SOURCEDIR/phresummed:\
+                                                        $SOURCEDIR/common" fonlllha
+mkdir -p $INSTALLROOT/bin;
+cp *.o fonlllha $INSTALLROOT/bin;
```

Comment:
Why copying the `*.o` in `bin`? Maybe you want to use something like `$INSTALLROOT/share` or similar?